src/rotest/cli/server.py

```diff
@@ -37,8 +38,11 @@ def start_server(server_port, run_django_server, django_port):
     try:
         if run_django_server:
             print("Running the Django server as well")
+            app_directory = os.path.dirname(search_config_file())
+            manage_py_location = os.path.join(app_directory, "manage.py")
             django_process = subprocess.Popen(
-                ["django-admin",
+                ["python",
```

Comment:
Works well for windows?

Reply:
Yes.
I actually worked on a Windows PC,
Also tox passed in Windows (except for the import order)